malariagen_data/anoph/cnv_data.py

```diff
@@ -189,6 +201,11 @@ def cnv_hmm(
                         inline_array=inline_array,
                         chunks=chunks,
                     )
+
+                    # If no CNV HMM dataset was found then skip
+                    if y is None:
+                        continue
+
                     ly.append(y)
 
```

Comment:
```suggestion

                    # If no CNV HMM dataset was found then skip
                    if y is None:
                        continue

                    ly.append(y)

                if len(ly) == 0:
                    # Bail out, no data for given sample sets and analysis.
                    raise ValueError(
                        f"No data found for requested sample sets."
                    )

```

Comparing with the `haplotypes()` function, there is an extra check here just in case there is no data at all for the requested sample sets.

Reply:
Thanks @alimanfoo , will do.